### DISCUSSION TITLE
Problem on grain growth during free sintering: non-uniform domain shrinkage with periodicity

### DISCUSSION THREAD

**Discussion by JingShuShi** [idaholab/moose#16860] [Q&A Modules: Phase field] [answered]
Dear Moose experts,

I want to study the sintering using phase field + tensor mechanics modules. The key outputs I am looking for are the volume shrinkage during sintering and evaluate the effective properties (e.g. elastic stress state) from the whole simulation domain.

My initial idea is to setup a RVE with periodic boundary conditions and randomly fill the grains to certain volume fraction (or import from existing packing file), then assign only one order parameter to solve the interface evolution. Then adding rigid body motion, and coupled to tensor mechanics to get the stress/strain fields.

If I now consider free sintering with gravity in y-direction, during sintering the pores are closed and grains will shrink in y-direction due to gravity, but the top boundary will still be the same as t=0, so the domain volume is constant, thus the effective volume fraction in the whole domain remains constant. Is it possible to set the domain to move and follow the shrinkage of top layer of grains?

I also have a second question on how to apply gravity onto the material:
option1: add it through force density in Rigid Body Motion on each grain (or maybe on elements with concentration > 0.95).
option2: add it as body force term into tensor mechanics module, so the extra strain energy from gravity will go into total free energy.

Could you please tell me which option would be better and why?
Thanks for the help.

Best Regards,
Hao

**Comment by SudiptaBiswas** [comment, marked as answer]
I believe we don't yet have the capability to change the domain size with the shrinking particles. You can use the RVE based analysis to apply load and capture the associated deformation behavior including volume/shape change. https://www.sciencedirect.com/science/article/pii/S0168874X20301165

If you want to model initial powder stacking, you can apply the gravitational force to each of the particles but you will need a way to detect contact between particles to avoid any overlap.

**Reply by JingShuShi** [reply]
Thanks for the paper, I had a look at it and got the idea of the global strain periodicity.  In this method, you need to specify the reference point that has no deformation. If the reference point is set at the center of the bottom boundary, then apply load on the top boundary, this will result only in top boundary moving, the displacements on top/bottom boundaries are not periodic but all the other quantities (stress, concentration, etc.) are still kept periodic?

For the initial powder stacking, by avoiding overlap, do you mean avoiding this only for the initial conditions, or during the whole simulation? Will the rigid body motion already ensure the particles without overlapping with each other?

**Reply by SudiptaBiswas** [reply]
For a periodic domain, I often consider the center of the domain as the reference point. With this, you would allow both the boundaries to deform and your stress-strain would be periodic. 

The whole simulation, if you apply external forces. Rigid body motion under externally applied force will not be sufficient for this.

**Reply by JingShuShi** [reply]
Thanks for the answer, I see, in order to maintain both stress and strain periodicity, the reference point has to be in the center, otherwise the final result will be asymmetric.

As you pointed out, if the external force is applied, the rigid body motion could not prevent the particles get into contact, then it is more like the pressurized sintering case, where the diffusion at interface will be enhanced by the interface pushed towards each other. I don't see the problem with interface getting close/overlap, will mass not be conserved in this case? What about the two centroids getting too close then? Will they merge into one centroid?

**Reply by SudiptaBiswas** [reply]
It would depend on how you apply the force and how you resolve the contact between particles. You just have to make sure nothing un-physical happens.

**Reply by JingShuShi** [reply]
thanks, I will try some simple test cases to see if any un-physical happens.
